test/test_torch.py

```diff
@@ -7317,7 +7317,7 @@ def test_logical(self):
                     torch.ne(x, b, out=boolRes)
                     self.assertEqual(byteRes.bool(), boolRes)
 
-                    self.assertEquals(len(warningsCount), 6)
+                    self.assertEquals(len(warningsCount), 5)
```

Comment:
What happened here? which warning went away?

Reply:
Based on conversation with @izdeby: we can remove warning: torch.lt received 'out' parameter with dtype torch.uint8, this behavior is now deprecated